packages/content/package.json

```diff
@@ -7,21 +7,36 @@
   "license": "Apache-2.0",
   "type": "module",
   "scripts": {
-    "build": "wireit"
+    "build:prod": "wireit",
+    "build:dev": "wireit"
   },
   "wireit": {
-    "build": {
-      "command": "eleventy --config=eleventy.config.cjs",
+    "build:prod": {
+      "command": "eleventy --config=eleventy.config.prod.cjs",
       "files": [
-        "site",
-        ".eleventy.cjs"
+        "eleventy.config.prod.cjs",
+        "site"
       ],
       "output": [
         "_site"
       ],
       "dependencies": [
-        "../components:build"
+        "../client:build:prod"
+      ]
+    },
+    "build:dev": {
+      "command": "eleventy --config=eleventy.config.dev.cjs",
```

Comment:
should this have a dependency on `../client:build:dev`?

Reply:
No, because otherwise every time the client code changes, Eleventy will re-run unnecessarily, because in dev mode that's just a symlink.

Reply:
Hmm... seems like that leaves a situation where the client output could be stale. Is there a different type of dependency at play here. Like: when you run task A, ensure B is up to date, but when you run B, do _not_ dirty A.